templates/events/show.html.twig

```diff
@@ -63,49 +63,6 @@
     </script>
 {% endblock %}
 
-{% block javascripts %}
-    <script src="https://maps.googleapis.com/maps/api/js?key={{ google_maps_api_key }}"></script>
-    <script type="text/javascript">
-        var map = null;
-        function initMap() {
-            var point = {
-                lat: {{ event.latitude }},
-                lng: {{ event.longitude }}
-            };
-
-            map = new google.maps.Map(document.getElementById('map'), {
-                center: point,
-                zoom: 15
-            });
-
-            var marker = new google.maps.Marker({
-                position: point,
-                map: map,
-                title: '{{ event.name|e('js') }}'
-            });
-        }
-        initMap();
-
-        Kernel.onLoad(function(di) {
-            var unregistrationButton = dom('.unregister-event');
-
-            unregistrationButton && on(unregistrationButton, 'click', function () {
```

Comment:
do we really want to remove the event unregistration process?

Reply:
oups. nope